website/.env.development

```
@@ -20,3 +20,4 @@ SENDGRID_SUPPRESSION_LIST_ID=45634
 SENDGRID_API_KEY="SG.Q****"
 FEATURE_ENABLE_NEW_WEBSITE="true"
 DATABASE_URL="postgresql://social-income:social-income@localhost:5432/social-income"
+STORYBLOK_MOCK_MODE="record"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)